src/test/java/core/basesyntax/service/impl/DataParserServiceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import core.basesyntax.service.FileReadService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class DataParserServiceImplTest {
+
+    private static final String INPUT_PATH = "src/test/resources/file_read_service_test.csv";
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+    private final FileReadService fileReadService = new FileReadServiceImpl();
+    private final DataParserService dataParserService = new DataParserServiceImpl();
+
+    @Test
+    public void dataParserService_ok() {
+        String data = fileReadService.readFromFile(INPUT_PATH);
```

Comment:
Maybe u don't have to mix to parseMethod some methods, which use reading from file
U can invoke your method on any string